ci/all-modules/hm.nix

```diff
@@ -0,0 +1,14 @@
+{
+  # keep-sorted start block=yes
+  i18n.inputMethod = {
+    enable = false;
+    type = "fcitx5";
+  };
+  programs = {
+
+  };
+  services = {
+
+  };
```

Comment:
I think defining these as empty attrsets requires some explanation. Assuming it actually _is_ necessary?